lib/reversetunnel/remotesite.go

```diff
@@ -697,10 +696,25 @@ func (s *remoteSite) getLocalWatchedCerts(remoteClusterVersion string) (types.Ce
 		}, nil
 	}
 
+	// Delete in 13.0.
```

Comment:
Use capitalized letters across delete notes

```suggestion
	// DELETE IN 13.0.0
```